client/app/deploy/services/loadBalancerService.js

```diff
@@ -0,0 +1,51 @@
+/* TODO: enable linting and fix resulting errors */
+/* eslint-disable */
+/* Copyright (c) Trainline Limited, 2016-2017. All rights reserved. See LICENSE.txt in the project root for license information. */
+
+'use strict';
+
+angular.module('EnvironmentManager.deploy').service('clientLoadBalancerService',
+  function ($http) {
+    var self = this;
+
+    self.dnsSuffix = 'service.ttlnonprod.local';
```

Comment:
This is trainline specific.

Reply:
How do we usually deal with things like this?